### ISSUE TITLE
Improve layout of social icons in footer of BOSC pages

### ISSUE THREAD

**Issue by nlharris** [OBF/homepage#84]
not pressing, but it would be nice if the icons didn't wrap to a second row.

<img width="971" alt="Screen Shot 2023-02-27 at 2 52 49 PM" src="https://user-images.githubusercontent.com/348704/221705806-26588f7b-6160-4f2e-af19-4a92b49c9085.png">

Karsten wrote:
```
In the Theme File Editor I found the following entry, which seems to control the arrangement of icons into five columns:

.networks {
    display: grid;
    grid-template-columns: 15% 15% 15% 15% 15%;
}

I assume there must be a reason for the network logos to sum up to 60%, so I changed it to 

.networks {
    display: grid;
    grid-template-columns: 10% 10% 10% 10% 10% 10%;
}

That worked for the OBF footer:
<image.png>
(unfortunately this only shows in Firefox - Chrome is still displaying the old version, must be cached)

There is a similar section for BOSC footers in the file css/style-bosc.css on GitHub:

.bosc-footer-sub-section .networks {
    display: grid;
    grid-template-columns: 18% 24% 18% 18% 18%;
}

Looks like the Twitter logo needs a bit of extra space due to the hashtag below it.
Not sure if it's important for the grid to sum up to 96%, but if so, this could work:

.bosc-footer-sub-section .networks {
    display: grid;
    grid-template-columns: 15% 21% 15% 15% 15% 15%;
}
```

**Comment by SrijanVerma04** [comment]
Hi @nlharris can I contribute to this issue.
That grid-template-columns : 15% 21% 15% 15% 15% 15% looks very accurate. I tried changing the display to inline-flex but that doesn't work as the twitter hashtag will be inline. 

Can i create a pull request with changing the grid percentage that will allign all the icons?

**Comment by peterjc** [comment]
Closed by #88
